provider/pkg/provider/provider.go

```diff
@@ -830,11 +830,11 @@ func (p *cfnProvider) Create(ctx context.Context, req *pulumirpc.CreateRequest)
 
 	resourceToken := string(urn.Type())
 	var id *string
-	var outputs map[string]interface{}
+	var outputs resource.PropertyMap
 	var createErr error
-	var payload map[string]interface{}
+	timeout := time.Duration(req.GetTimeout()) * time.Second
```

Comment:
Unlike bridged providers, guessing there is no need to merge with schema-derived timeouts here?

Also, double-checking, this will be `0` if req.GetTimeout() is unset?

Reply:
No, there's no notion of passing timeouts to the CloudControl based resources (yet).

At the current time no resource is using timeouts yet. This is just prep for the next changes related to CFN CustomResources

Reply:
And yes, this will be `0` if no timeouts were specified